[Form] Explain `choice_label` option for enums (#19324)
symfony/symfony-docs: reference/forms/types/enum.rst

        }
    }

Another way of controlling the label displayed in the ``<option>`` element is by

> using the `choice_label`. In your `Enum` add a label method::

Reviewer: ```suggestion
using the ``choice_label`` option. Add a ``label`` method to your enum::
```

    // src/Enum/TexAlign.php
    // ...
    
    public function label(): string
    {
        return match ($this) {
            self::Left => 'Left aligned',

>             self::Center => 'Center aligned',

Reviewer: ```suggestion
```

lets remove center to ease and shorten the example

            self::Right => 'Right aligned',
        };
    }

> Then in your `EnumType` add the `choice_label`::

Reviewer: ```suggestion
Use your ``label`` method accordingly::
```
